packages/rgbpp/src/rgbpp/types.ts

```diff
@@ -0,0 +1,24 @@
+import { DataSource } from '@rgbpp-sdk/btc';
+import { BtcTransferVirtualTxResult, Collector, Hex } from '@rgbpp-sdk/ckb';
+
+export interface RgbppTransferTxParams {
+  // The collector that collects CKB live cells and transactions
+  collector: Collector;
+  // The transferred RGB++ xUDT type script args
+  xudtTypeArgs: Hex;
+  // The rgbpp assets cell lock script args array whose data structure is: out_index | bitcoin_tx_id
+  rgbppLockArgsList: Hex[];
+  // The XUDT amount to be transferred, if the noMergeOutputCells is true, the transferAmount will be ignored
+  transferAmount: bigint;
+  // The sender BTC address
+  fromBtcAddress: string;
+  // The receiver BTC address
+  toBtcAddress: string;
+  btcDataSource: DataSource;
+  isMainnet: boolean;
+}
+
+export interface RgbppTransferTxResult {
+  ckbVirtualTxResult: BtcTransferVirtualTxResult;
+  btcTxHexToSign: Hex;
```

Comment:
I think `btcTxHexToSign` should be renamed to `btcPsbtHexToSign` or, more simply, `btcPsbtHex`.

Reply:
Make sense for me

Reply:
https://github.com/ckb-cell/rgbpp-sdk/pull/216/commits/8ccd765f4ee750f2a0ee73d84a3100cd73a2a77d